llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -940,6 +940,18 @@ static void simplifyRecipe(VPRecipeBase &R, VPTypeAnalysis &TypeInfo) {
 #endif
   }
 
+  VPValue *B;
+  VPValue *C;
+  VPValue *D;
+  // Simplify (X && Y) || (X && !Y) -> X.
```

Comment:
Keep comments and code consistent: have both use A,B,C,D or both X,Y,Z.

Reply:
How about keeping the description (X && Y) || (X && !Y) -> X and use X, Y, X1, Y1 instead of A, B, C, D in the code, respectively?